app/styles/partials/speaker.scss

```diff
@@ -14,3 +14,9 @@
     }
   }
 }
+
+.speaker-description {
+  word-wrap: break-word;
+  height: 240px;
```

Comment:
Don't fix the height like this. Instead, use `max-height`. And also, see how this is implemented in the current site. -> https://eventyay.com/e/5642d9a1/ . We get a scroll bar on hover if the speaker description is too big. We'd like the same feature here too. 

Our aim, is to first get the all the feature of the current UI into the new frontend.